eng/pipelines/common/platform-matrix.yml

```diff
@@ -25,6 +25,7 @@ parameters:
 jobs:
 
 # Linux arm
+
```

Comment:
I don't see any reason for these formatting changes, can we revert changes to this file?

Reply:
I think it looks much better to have the platform comments like some kind of separators. Without those, they get a bit lost with the rest of the code. I think it's more readable this way in this particular case.

Reply:
OK, sounds good.